### ISSUE TITLE
Custom Websites Don't Launch

### ISSUE THREAD

**Issue by CardcaptorRLH85** [moraroy/NonSteamLaunchers-On-Steam-Deck#242]
When I add a custom website it doesn't launch.  I click "Play" and nothing happens.  This happens in both formats I can use for the site in question "www.github.com" and "https://www.github.com" (the sites I'm trying to access are of the format xyz.domain.tld).

Am I doing something wrong here?  I noticed that the launch options are running Chrome.  Is it running the Steam internal Chromium build or do I have to have Chrome installed?  I'm only running Firefox on my Steam Deck right now.

**Comment by moraroy** [comment]
Yeah you need Google Chrome installed, also you can just put github.com no need for www

**Comment by CardcaptorRLH85** [comment]
@moraroy I was only using Github as an example, I have a web game that I'm hosting at a personal domain that is in the form xyz.my-domain.tld.  I just didn't want to post a domain hosted from my home internet publicly.

**Comment by moraroy** [comment]
@CardcaptorRLH85 oh ok..OK... what you could do if it doesn't work initially..... Is Create a dummy shortcut like input in the text field... google.com... and then in the shortcut that is made edit the launch options.. and input your actual url that you want. Make sure Chrome is installed first though

**Comment by CardcaptorRLH85** [comment]
@moraroy I'm still seeing the same issue both before and after editing the launch options.  Chrome is installed from Flatpak on my deck and when I click Play it flashes to cancel then stop and then back to play as though it had just crashed (or closed).

**Comment by moraroy** [comment]
@CardcaptorRLH85 are you forcing comptabability? Make sure it's unchecked and check your url in the launchoptions no reason why it wouldn't work.. make sure chrome is functioning and working on its own

**Comment by CardcaptorRLH85** [comment]
@moraroy Chrome is running by itself and everything looks right in the launch options.  Here's my launch options line `run --branch=stable --arch=x86_64 --command=/app/bin/chrome --file-forwarding com.google.Chrome @@u @@ --window-size=1280,800 --force-device-scale-factor=1.00 --device-scale-factor=1.00 --kiosk https://github.com/ --chrome-kiosk-type=fullscreen --no-first-run --enable-features=OverlayScrollbar`

**Comment by moraroy** [comment]
@CardcaptorRLH85 yeah that's the issue then you need to install chrome via the Discover Store on arch linux

**Comment by CardcaptorRLH85** [comment]
@moraroy That's strange, I **did** install from the discover store.  Pacman isn't currently useable on the Steam Deck without modifying system files.

**Comment by moraroy** [comment]
@CardcaptorRLH85 wow that is weird, maybe Chrome is no longer being installed there? This may be a new problem..

**Comment by CardcaptorRLH85** [comment]
@moraroy I see the com.google.Chrome folder in /home/deck/.var  Is this where it should be to work with NonSteamLaunchers?

**Comment by CardcaptorRLH85** [comment]
@moraroy Sorry for the double-post, it looks like we posted our last comments at about the same time.  I've tried changing the Target and Start In based on the location of Chrome on my deck. It's still not running. Then I tried to change the command since it still mentions /app/bin/chrome and still got nothing. I'm not sure what has to change for this feature to work again but it looks like either SteamOS or Discover has moved where Flatpak apps are stored and it has broken your web shortcuts.

**Comment by moraroy** [comment]
@CardcaptorRLH85 interedting...is the install directory moved just for chrome? Or does this happen to any new flat packs now installed?

**Comment by CardcaptorRLH85** [comment]
@moraroy as far as I can tell, all flatpak apps are in the new location. However just ~~Chromium based apps~~ Chromium and Chrome fail to load. (Edge actually loads just fine.)
